demo/Demo Instructions.md

```diff
@@ -0,0 +1,21 @@
+# Pelorus Demo Instructions
```

Comment:
please move this file to `docs/Demo.md`

Also, this file should actually include an explanation of what's happening along the way. Think of it as a quickstart that one would walk through to understand how Pelorus works.

Reply:
Updated. Could use a second lookover